sdk/identity/azure-identity/CMakeSettings.json

```diff
@@ -0,0 +1,44 @@
+﻿{
+  "configurations": [
```

Comment:
Committing this file breaks other developers.
As a Developer, I can set up my VIsual Studio settings with some configurations I want to use where I set this and that option.
Then, after getting last master, It would break my configuration.

I insist that pushing this file ends up to be more problematic.
If the intention is for first-time-users to open Visual Studio and get some initial configuration, that will work, but later, once they learn more and start changing that configuration, will they commit changes to this file?  

Also, This changes are not required at all for vcpkg support.
If you feel strong about sharing this configuration, please move it to another PR